package.json

```diff
@@ -34,7 +34,8 @@
     "build:libs": "npm-run-all -p ui:build icons:build core:build -s angular:build schematics:build",
     "build:apps": "npm-run-all -p angular:dev:build core:dev:build",
     "build:website": "yarn --cwd apps/website run build",
-    "build:storybook:angular": "yarn --cwd packages/angular run build-storybook",
+    "build:storybook:angular": "yarn --cwd packages/angular run build-storybook && npx -p @storybook/cli@6.0.0-rc.26 sb extract dist/website/storybook/angular",
+    "postbuild:storybook:angular": "node scripts/website/generate-sw",
```

Comment:
After the website and stoarybook/angular apps are built the service worker is generated based on the contents of `dist/website` directory.